Source/Core/DolphinQt2/Settings.h

```diff
@@ -51,4 +51,18 @@ class Settings final : public QSettings
   bool GetRenderToMain() const;
   bool GetFullScreen() const;
   QSize GetRenderWindowSize() const;
+
+  // Columns
+  bool& IsBannerVisible() const;
```

Comment:
That would greatly complicate the code as I'm capturing this boolean in a lambda. I think that I would have to write a function for every column if I didn't have those references.